TrainingApp/StartViewController.swift

```diff
@@ -14,15 +14,23 @@ class StartViewController: UIViewController {
     @IBOutlet private var signInButton: UIButton!
     @IBOutlet private var signUpButton: UIButton!
     
+    let userDefaults = UserDefaults.standard
     // MARK: - Lifecycle
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         self.navigationController?.navigationBar.isHidden = false
+        checkLanguage()
         configureUI()
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        checkLanguage()
```

Comment:
why we use checkLanguage() in viewWillAppear and viewDidLoad ?

Reply:
I've removed it. Localization seems to work without it. 👍